### PR TITLE
add serverless plugin for automated deploy

### PR DESCRIPTION
**Pull request by andrey-naumkyn**
Service is done.
S3 - http://my-second-store-app.s3-website-eu-west-1.amazonaws.com/
Website - https://d2jybe0jhu4tc2.cloudfront.net

### CODE REVIEW

File: serverless-single-page-app-plugin/package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (juanrivera32) [inline serverless-single-page-app-plugin/package.json:1]: What is this file being used for?

File: serverless-single-page-app-plugin/index.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+'use strict';
```
Comment (juanrivera32) [inline serverless-single-page-app-plugin/index.js:1]: How is this file different from the one located at `.serverless_plugins/serverless-single-page-app-plugin.js`

**Review by juanrivera32 — COMMENTED** [review]
URLs working as expected. Config changes added correctly. I only have these two questions

File: serverless-single-page-app-plugin/package.json (hunk 1 above)
Comment (andrey-naumkyn) [reply serverless-single-page-app-plugin/package.json]: hey Juan, haven't noticed that we had initially this plugin in our repo. Remove duplicate.

File: serverless-single-page-app-plugin/index.js (hunk 2 above)
Comment (andrey-naumkyn) [reply serverless-single-page-app-plugin/index.js]: removed duplicate

**Pull request merged**